Makefile

```diff
@@ -120,6 +120,10 @@ static: static-dependencies
 release-static: static
         # TODO(unassigned): cmake release flags + optimizations/hardening when we're out of alpha
 
+release-static-android: static-dependencies
+	mkdir -p $(build)
+	cd $(build) && $(cmake) $(cmake-debug) $(cmake-static) -D ANDROID=1 -D KOVRI_DATA_PATH="/data/local/tmp/.kovri" ../ && $(MAKE)
```

Comment:
Did you need to manually copy resources to this data path or did `make install-resources` somehow get sent there (is `~/.kovri` symlinked)?